jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -350,3 +352,75 @@ def test_correct_output_spectral_y_units(cubeviz_helper, spectrum1d_cube_custom_
 
     mm.calculate_moment()
     assert mm.moment.unit == moment_unit.replace('m', 'um')
+
+
+@pytest.mark.parametrize("flux_angle_unit", [(u.Unit(x), u.sr) for x in SPEC_PHOTON_FLUX_DENSITY_UNITS] + [(u.Unit(x), PIX2) for x in SPEC_PHOTON_FLUX_DENSITY_UNITS])  # noqa
```

Comment:
Maybe less confusing to use nested parametrization here? This is a bit hard to read.

https://stackoverflow.com/questions/40164910/nested-parametrized-tests-pytest

Reply:
I also wonder if this is more effective as a test *class* where the moment map is pre-calculated and re-used in a test method via parametrization, instead of recalculating for each combo. Does that make sense?